src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/response/SearchStaffUserResponse.java

```diff
@@ -17,7 +17,7 @@
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
+@SuperBuilder
```

Comment:
Please check the mutation coverage for this annotation - it may require additional tests.

Reply:
Its used to inheritance so additional test case is not required

Reply:
Please do a verification check as I can see that it has other uses that might not be covered by our tests but will be mutation tested:

_@SuperBuilder can generate so-called 'singular' methods for collection parameters/fields. For details, see [the @Singular documentation in @Builder](https://projectlombok.org/features/Builder#singular).

@SuperBuilder generates a protected constructor on the class that takes a builder instance as a parameter. This constructor sets the fields of the new instance to the values from the builder._

Reply:
Removed SuperBuilder